### DISCUSSION TITLE
Welcome to Azure IPAM Discussions!

### DISCUSSION THREAD

**Discussion by CloudViking** [Azure/ipam#1] [Announcements]
## 👋 Welcome to Azure IPAM!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by satyapalav** [comment]
Hi Team,
I am looking to deploy this solution on private end points and url must be accessed only internally.
Please help me with step by step approach or release separate deploy for internal access.

**Reply by DCMattyG** [reply]
Hello @satyapalav, and thank you so much for reaching out in our Azure IPAM Discussions!

For the Azure IPAM Deployment script which we've included in the repository, we have a few customizations that can be done (via switches), but I think you can agree that it's not possible to take into consideration every possible way to deploy this solution in Azure. That said, we've picked a few methods that we can fully automate and built those into the script.

For other deployments, such as one using all Private Endpoints like you've mentioned, we do have numerous customers whom have chosen to deploy the Azure IPAM application this way. There are quite a few different ways to do this, so perhaps I can list a few options for you, and you can tell me which one aligns best to the way you'd prefer to secure access to Azure IPAM, and then I can help guide you down that path.

Most common ways customers are privately integrating Azure IPAM:
1. App Services w/ Private Endpoints + Application Gateway
2. Azure Kubernetes Service w/ Ingress (such as NGINX)
3. Azure Container Apps (this is tricky, but it can be done)

Do any of those align well to how you'd prefer to privately integrate Azure IPAM into your Azure network?
